types/global.d.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
+  // Using `namespace` here is okay because this is how the Jest types are
+  // defined.
+  namespace jest {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/ban-types
+    interface Matchers<R, T = {}> {
```

Comment:
I believe technically we can drop the `T = {}`, according to the [docs](https://jestjs.io/docs/27.x/expect#expectextendmatchers).

Reply:
Great! One less ESLint ignore comment now as well. Done in https://github.com/MetaMask/eth-json-rpc-middleware/pull/191/commits/e9f4fc5e3c4c5ebeb2ec2602d49a79abf80edfdc

Reply:
I have made the equivalent change on core as well: https://github.com/MetaMask/core/pull/1099